src/modules/todoClass.js

```diff
@@ -67,4 +66,51 @@ export const editBtn = () => {
 
 }
 
+export const deleteDone = (e) => {
+  const checked = e.target.closest('.check-box');
+  if(!checked) return;
+  const cleared = parseInt(checked.getAttribute('data-id'), 10);
+  const task = JSON.parse(localStorage.getItem('todos')) || [];
+  const find = task.find((todo) => todo.id === cleared);
+  find.completed = !find.completed;
+ 
+  localStorage.setItem('todos', JSON.stringify(task));
+};
+
+
+
+
+export const clear = (e) => {
+  const set = JSON.parse(localStorage.getItem('todos')) || [];
+  const notDone = set.filter((list) => !list.completed);
+  
+
+  if (notDone.length === 0){
+    generatedList.style.display = 'none';
+  }
+  
+  const todoLists = document.querySelector('.todo-lists');
+
+  for (let i = 0; i < set.length; i++){
+    
+    if(set[i].completed === true){
+      //n = document.getElementById(`${set[i].id}`);
```

Comment:
- [ ] Kindly remove all commented out lines of code to make your code clean :recycle: